scripts/route_check.py

```diff
@@ -585,6 +585,9 @@ def get_soc_ips(config_db):
         if mux_entry.get("cable_type", "") == "active-active" and "soc_ipv4" in mux_entry:
             soc_ips.append(mux_entry["soc_ipv4"])
 
+            if "soc_ipv6" in mux_entry and mux_entry["soc_ipv6"]:
```

Comment:
Placing the check here will make `soc_ipv6` dependent on `soc_ipv4` presence, better to check `soc_ipv4` and `soc_ipv6` separately.

Reply:
> Placing the check here will make `soc_ipv6` dependent on `soc_ipv4` presence, better to check `soc_ipv4` and `soc_ipv6` separately.

Addressed.